src/rtrc.js

```diff
@@ -623,6 +627,82 @@
 		$('#krRTRC_loader').hide();
 	}
 
+	function applyOresAnnotations($feedContent, callback) {
+		var revids;
+
+		// Find all revids names inside the feed
+		revids = [];
+		$feedContent.filter('.mw-rtrc-item').each(function () {
```

Comment:
Using `$.map()` would simplify this code:

```js
revids = $.map($feedContent.filter('.mw-rtrc-item'), function (node) {
  return $(node).attr('data-diff');
});
```
map() automatically filters out undefined/null from attr().